site/static/compare/script.js

```diff
@@ -448,6 +466,12 @@ app.mixin({
             }
             return klass;
         },
+        signIfPositive(pct) {
```

Comment:
Can this be removed? It doesn't look like it's being used anymore.

Reply:
That's actually an oversight on my part, I removed it unadvertedly. I think that the + was useful visually, so I will revert it back, if you don't mind.